bodhi/server/services/updates.py

```diff
@@ -18,6 +18,7 @@
 from cornice import Service
 from sqlalchemy import func, distinct
 from sqlalchemy.sql import or_
+import transaction
```

Comment:
Surprisingly, it is! It's a weird system. I've had quite a time learning how sqlalchemy, zope.sqlalchemy, and pyramid all come together to do transactions. They seem to use this transaction library to "register" transaction events and transaction handlers.

```
$ dnf provides /usr/lib/python2.7/site-packages/transaction
python-transaction-1.6.1-2.fc25.noarch : Transaction management for Python
```